### PR TITLE
Menu: Improve a11y of search by adding labels

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

When we implemented all the fancy CSS for the new search input, we forgot to migrate readable labels, as @ScottPainterGNM [pointed out](https://docs.google.com/document/d/1Hx93Ytr0FaUHYpU3SSNRaY3KCY_cd9gUSj1AC1iY55s/edit?ts=596c996e) (See 4).

## What is the value of this and can you measure success?

Better a11y.

## Does this affect other platforms - Amp, Apps, etc?

No.

## Screenshots

No.

## Tested in CODE?

No.

### CODE REVIEW

File: common/app/views/fragments/nav/newHeaderMenu.scala.html [hunk 1]
```diff
@@ -92,12 +89,13 @@
                                 for="submit-google-search"
                                 type="submit">
                             <i class="right-arrow__icon"></i>
+                            <span class="u-h">Submit your search</span>
```
Comment (NataliaLKB) [inline common/app/views/fragments/nav/newHeaderMenu.scala.html]: What about changing text to "Search with Google". I remember @zeftilldeath and Gustavo talking about this last week

**Review by NataliaLKB — APPROVED** [review]
Code changes look good to me. Only one question about further changes we could make.

File: common/app/views/fragments/nav/newHeaderMenu.scala.html (hunk 1 above)
Comment (ScottPainterGNM) [inline common/app/views/fragments/nav/newHeaderMenu.scala.html]: I think that's a good shout, nice and clear about what to expect from the form.

**Review by scottp719 — APPROVED** [review]
I agree with Natalia's comment but otherwise looking good 👍

**Pull request merged**